baselines/dom.generated.d.ts

```diff
@@ -7536,16 +7536,12 @@ declare var HashChangeEvent: {
 interface History {
     readonly length: number;
     readonly state: any;
-    back(distance?: any): void;
-    forward(distance?: any): void;
-    go(delta?: any): void;
-    pushState(statedata: any, title?: string, url?: string): void;
-    replaceState(statedata: any, title?: string, url?: string): void;
-}
-
-declare var History: {
-    prototype: History;
```

Comment:
The `History` is the prototype for the type. the constructor can not be called directly, but that is the same as the rest of the HTML*Element classes as well. so we should keep both a type and a value for `History`. we already have a value `history: History` on `Window` and the global object.